### ISSUE TITLE
nested message payload not correctly serialized

### ISSUE THREAD

**Issue by eelkevdbos** [cleentfaar/slack#20]
**method used**: chat.postMessage

When I try to post a message with attachments, the attachments are not 'seen' by the slack service. I tried encoding them to json prior to sending them to slack via a guzzle event listener like this:

```php
$guzzle->getEmitter()->on('before', function (BeforeEvent $before) {
    $body = $before->getRequest()->getBody();
    $attachments = $body->getField('attachments');
    $body->setField('attachments', json_encode($attachments));
});
```

And got what I expected (a chat message with attachments). However, this leads me to think there is something wrong with the serialization process of nested payload entries. 

**In short, my findings**:
```php
//does not work
[
    'attachments' => [
        ['title' => 'hello world']
    ]
]

//does work
[
    'attachments' => "[{'title': 'hello world'}]"
]
```

**Comment by agentsib** [comment]
Wating approve #14

**Comment by cleentfaar** [comment]
@eelkevdbos Sorry to keep you waiting so long, the PR has been merged. I will be making a new release soon; you can use the master version in the meantime.
